types/espree/index.d.ts

```diff
@@ -23,6 +25,7 @@ declare module "espree" {
       | 2020
       | 2021
       | 2022
+      | 2023
       | "latest";
```

Comment:
We'll never use other value, we can remove them all except `"latest"`.

Reply:
It sounds good

Reply:
[075d9cb](https://github.com/prettier/prettier/pull/13644/commits/075d9cbe68421040e7c1c6903d76ac34683c622f)